include/swift/SwiftRemoteMirror/SwiftRemoteMirror.h

```diff
@@ -80,6 +80,12 @@ void
 swift_reflection_addReflectionInfo(SwiftReflectionContextRef ContextRef,
                                    swift_reflection_info_t Info);
 
+/// Add reflection sections that may not be contiguous from a loaded Swift image.
```

Comment:
@vedantk @adrian-prantl @compnerd Is this comment ok? Should I add something explaining we use this outside of macOS?

Reply:
This comment looks good to me as-is!